modules/db_instance/outputs.tf

```
@@ -28,6 +28,16 @@ output "db_instance_endpoint" {
   value       = try(aws_db_instance.this[0].endpoint, "")
 }
 
+output "db_instance_engine" {
+  description = "The RDS engine type"
```

from the provider docs
```suggestion
  description = "The database engine"
```